cluster/gce/windows/k8s-node-setup.psm1

```diff
@@ -953,18 +953,17 @@ function Configure-CniNetworking {
 #   CLUSTER_IP_RANGE
 #   SERVICE_CLUSTER_IP_RANGE
 function Configure_Dockerd_CniNetworking {
-  $CNI_RELEASE_VERSION = 'v0.8.2-gke.0'
+  $CNI_RELEASE_VERSION = 'v0.8.5'
```

Comment:
this powershell module is used soley by GCE/GKE, please keep using the gs bucket (gs://gke-release/cni-plugins/v0.8.5-gke.0/) that our release team maintains.

Reply:
@yliaog -- Is there a reason we'd want the Windows tarballs to diverge from the ones provided by the CNI plugins maintainers & Kubernetes community?

The goal here is to provide everything from a single bucket, which has shared control between the CNI maintainers and Kubernetes Release Engineering.

Are there concerns on the Windows side that make this vary from the artifacts we're retrieving on Linux side (which also get used for the GCE/GKE tests)?

cc: @thockin

Reply:
(That is to say, I think all of these artifacts should come from the same place and I don't think this PR should be held.)

Reply:
i'm going to create a PR to make them env variables

Reply:
@yliaog -- Sounds good. I'm going to unhold this then!
Would you mind tagging me on the PR when you make it?

/hold cancel

Reply:
sure, i'll cc you the PR. thanks.